appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+# Test against the latest version of this Node.js version
+# environment:
+#   nodejs_version: "6"
+
+platform:
+  - x64
+
+cache:
+  - node_modules
+  - '%APPDATA%\npm-cache'
+
+init:
+  - git config --global core.autocrlf input
+
+# Install scripts (runs after repo cloning)
+install:
+  - ps: Install-Product node 6 x64
+  # - git reset --hard HEAD
+  - npm install
+  - npm prune
+
+build_script:
+  - node --version
+  - npm --version
```

Comment:
This is where I’ve seen these configs set in most of the examples I’ve seen, e.g., nodecg/dashboard (https://github.com/nodecg/dashboard/blob/master/appveyor.yml) -- though occasionally in the `test_script` section where there was no `build_script`, e.g., in the AppVeyor doc “Testing with Node.js and io.js” (https://www.appveyor.com/docs/lang/nodejs-iojs/).  I don’t recall seeing this set in the `install` section.